Sources/Bot/Services/MergeService.swift

```diff
@@ -190,7 +192,8 @@ extension MergeService {
                 else { return .value(.integrationDidChangeStatus(.done, metadata)) }
 
             switch metadata.mergeState {
-            case .clean:
+            case .clean,
+                 .unstable where !requiresAllStatusChecks:
```

Comment:
what does `unstable` mean?

Reply:
This is documented as a proper doc-comment on the `enum case` itself (i.e. you can alt-click on it to see quick help).

```
        /// Failing/pending commit status that is not part of the required status checks. Merging is still allowed.
        case unstable
```